modules/db_text/dbt_file.c

```diff
@@ -125,6 +126,12 @@ dbt_table_p dbt_load_file(const str *tbn, const str *dbn)
 	if(!fin)
 		return NULL;
 
+	buf = pkg_malloc(buffer_size);
```

Comment:
@ovidiusas , wouldn't be worthy to have this `buf` as static var in the function, so we can allocate it upon first usage and keep re-use it for the whole runtime? Instead of keep allocating and freeing it after each load....

Reply:
agreed, point taken